runtime/common/src/mmr.rs

```diff
@@ -140,13 +148,13 @@ impl<T: Config> pallet_session::OneSessionHandler<T::AccountId> for Module<T> wh
 	fn on_genesis_session<'a, I: 'a>(_validators: I) where
 		I: Iterator<Item=(&'a T::AccountId, Self::Key)>, Self::Key: 'a
 	{
-		Self::update_beefy_authorities();
+		Self::update_beefy_next_authorities();
```

Comment:
Genesis session should probably set current authorities as opposed to next

Reply:
We currently make the assumption that the first two session have the same authorities. This is enforced by the session pallet even though its interfaces don't communicate this invariant.